api_docgen/Makefile

```diff
@@ -16,7 +16,6 @@ ROOTDIR = ..
 -include $(ROOTDIR)/Makefile.build_config
 
 ifeq ($(DOCUMENTATION_TOOL),odoc)
-  include odoc/Makefile
```

Comment:
This breaks the build when `odoc` is set as the default configuration tool with `./configure --with-odoc`.

Reply:
I've changed it so now `include` is used and the calls from the main `Makefile` still work.
The problem with my change is that the destination of the built documentation changes to `api_docgen/ocamldoc/build/html`. Is it a problem ?

Reply:
Sorry for the delay. I think I managed to fix the mess I made in the manual :)